ui/components/app/confirm/footer/footer.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Footer = () => <div>CONFIRM FOOTER COMPONENT</div>;
+
+export default Footer;
```

Comment:
I understand these files are intended to be changed soon. just noting it might be nice to add todo comments with a quick mention of these files in case anyone comes across them.